From 7b5cb1228c914b38cbcd935be1bd419754636072 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 22 Sep 2015 18:19:04 +0800
Subject: [PATCH] soup-session.c: Fix Build on MinGW

MinGW requires that a prototype be declared for DllMain() before we can use
it, so define it here to fix the build, as the previous patch to add this
did not check for this part.
---
 libsoup/soup-session.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libsoup/soup-session.c b/libsoup/soup-session.c
index ca8d80c..1eda025 100644
--- a/libsoup/soup-session.c
+++ b/libsoup/soup-session.c
@@ -81,6 +81,10 @@
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 
+BOOL WINAPI DllMain (HINSTANCE hinstDLL,
+                     DWORD     fdwReason,
+                     LPVOID    lpvReserved);
+
 HMODULE soup_dll;
 
 BOOL WINAPI
-- 
2.3.5.windows.8
